site/content/docs/5.3/utilities/link.md

```diff
@@ -87,14 +87,20 @@ Just like the `.link-opacity-*-hover` utilities, `.link-offset` and `.link-under
 
 [Colored link helpers]({{< docsref "/helpers/colored-links/" >}}) have been updated to pair with our link utilities. Use the new utilities to modify the link opacity, underline opacity, and underline offset.
 
-<!--Boosted mod: feature limited to primary and light, we don't loop over "theme-colors"-->
+<!--Boosted mod: add bg-dark on the links which need it to ensure contrast-->
 {{< example >}}
-<p><a href="#" class="link-primary link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Primary link</a></p>
-<p><a href="#" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Light link</a></p>
+{{< colored-links.inline >}}
+{{- range (index $.Site.Data "theme-colors") }}
+<p><a href="#" class="link-{{ .name }} {{ if or (eq .name "primary") (eq .name "success") (eq .name "warning") (eq .name "info") (eq .name "light") -}}bg-dark{{- end }} link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">{{ .name | title }} link</a></p>
```

Comment:
Can't we use the same trick as Bootstrap, meaning the `color-contrast` field in `theme-colors.yml`